5.0.2 & 4.6.24 releases (#2898)
ibexa/documentation-developer: docs/release_notes/ibexa_dxp_v4.6.md

<div class="release-notes" markdown="1">

[[% set version = 'v4.6.24' %]]

[[= release_note_entry_begin("Collaboration ", '2025-09-09', ['Headless', 'Experience', 'Commerce', 'LTS Update', 'New feature']) =]]

#### Collaboration

> The new [Collaborative editing](collaborative_editing_guide.md) LTS Update allows multiple users to preview, review, and edit the same content, improving teamwork and streamlining the review process.

Reviewer: ```suggestion
The new [Collaborative editing](https://doc.ibexa.co/en/4.6/content_management/collaborative_editing/collaborative_editing_guide/) LTS Update allows multiple users to preview, review, and edit the same content, improving teamwork and streamlining the review process.
```
I'd use the absolute link, so that it points to the correct version always.

[[= release_note_entry_end() =]]

[[= release_note_entry_begin("AI Actions " + version, '2025-09-09', ['LTS Update']) =]]

#### Chat GPT 5.0 support

With improved reasoning and greater accuracy in mind, the AI Connector package has been enhanced by adding ChatGPT 5.0 to its list of supported LLMs.

![ChatGPT 5.0 on a list of supported LLMs](502_ai_connector_gpt_50.png "ChatGPT 5.0 on a list of supported LLMs")

[[= release_note_entry_end() =]]

[[= release_note_entry_begin("Discounts " + version, '2025-09-09', ['LTS Update', 'Commerce']) =]]

#### Discount indexing

Discounts now allow scheduling a re-indexing of discounted product catalog prices at the most convenient time by using the Ibexa Messenger package.
Ibexa Messenger is a customization of the Symfony Messenger package, created to adjust it to [[= product_name =]]'s needs.

Once properly configured, it uses a background queue to trigger price re-indexing, ensuring efficient use of system resources without causing performance disruptions.

##### PHP API

The following additions were made to the Discounts PHP API:

??? note "Events"

>     - [`Ibexa\Contracts\AdminUi\Event\ResolveVersionPreviewUrlEvent`](/api/php_api/php_api_reference/classes/Ibexa-Contracts-AdminUi-Event-ResolveVersionPreviewUrlEvent.html)

Reviewer: Same here, please use absolute links - otherwise the build will start failing if the class is removed in v5.

Just search and replace and prefix with https://doc.ibexa.co/en/4.6/ for 4.6 and (https://doc.ibexa.co/en/5.0 for v5

??? note "Search criteria"
    - [`Ibexa\Contracts\Discounts\Value\Query\Criterion\IndexedAtCriterion`](/api/php_api/php_api_reference/classes/Ibexa-Contracts-Discounts-Value-Query-Criterion-IndexedAtCriterion.html)
    - [`Ibexa\Contracts\Discounts\Value\Query\Criterion\UpdatedAtCriterion`](/api/php_api/php_api_reference/classes/Ibexa-Contracts-Discounts-Value-Query-Criterion-UpdatedAtCriterion.html)

[[= release_note_entry_end() =]]

[[= release_note_entry_begin("Ibexa DXP " + version, '2025-09-09', ['Headless', 'Experience', 'Commerce', 'New feature']) =]]

#### Improvements to notifications

An improved notifications system is now more intuitive.
Developers can now create and configure their own notification types, while users can now browse through a list of notifications, where they can either act on them or dismiss them.

![A searchable notifications list](502_notifications_screen.png "A searchable notifications list")

#### Developer experience

##### New packages

The only package that has been introduced in [[= product_name =]] v4.6.24 is ibexa/messenger.

##### New version of PHP Storm Plugin

> To further improve your experience with Ibexa DXP, a 1.14.0 version of PHP Storm Plugin has been released, which brings the following changes:

Reviewer: Please add link to https://ez-systems-developer-documentation--2898.com.readthedocs.build/en/2898/resources/phpstorm_plugin/

- Added support for Ibexa DXP v5.0
- Added compatibility with PhpStorm 2024.3.6+
- Added file template for Twig Component class
- Added code completion for Twig Component Groups in YAML config files and AsTwigComponent attribute
- Added code completion for Twig Component Types in YAML config files

##### Infrastructure

> - Redis 7.2+ are now [officially supported](https://doc.ibexa.co/en/4.6/getting_started/requirements/)

Reviewer: ```suggestion
- Redis 7.2+ is now [officially supported](https://doc.ibexa.co/en/4.6/getting_started/requirements/)
```

<div class="release-notes" markdown="1">

[[% set version = 'v4.6.24' %]]

> [[= release_note_entry_begin("Collaboration ", '2025-08-05', ['LTS Update', 'Commerce']) =]]

Reviewer: ```suggestion
[[= release_note_entry_begin("Collaboration ", '2025-X-X', ['Headless', 'Experience', 'Commerce', 'LTS Update', 'New feature') =]]
```
Collaboration is not limited to Commerce, it's available since Headless

And the date is invalid

#### Collaboration

> The new Collaborative editing feature allows multiple users to preview, review, and edit the same content or product, improving teamwork and streamlining the review process.

Reviewer 2: ```suggestion
The new Collaborative editing LTS update allows multiple users to preview, review, and edit the same content or product, improving teamwork and streamlining the review process.
```

> The new Collaborative editing feature allows multiple users to preview, review, and edit the same content or product, improving teamwork and streamlining the review process.

Reviewer: ```suggestion
The new Collaborative editing feature allows multiple users to preview, review, and edit the same content, improving teamwork and streamlining the review process.
```
See https://github.com/ibexa/documentation-developer/pull/2715#issuecomment-3284392933

Internal and external users can be invited to a collaboration session, through different sharing options.

With Real-time editing, more advanced part of the feature, users can see each other’s changes in the real time, or work on the content asynchronously.

Additionally, shared drafts can be accessed and managed through new dashboard tabs: **My shared drafts** and **Drafts shared with me**, helping users stay organized. 

[[= release_note_entry_end() =]]

> [[= release_note_entry_begin("AI Actions " + version, '2025-04-09', ['LTS Update']) =]]

Reviewer: Wrong date

#### Chat GPT 5.0 support

With improved reasoning and greater accuracy in mind, the AI Connector package has been enhanced by adding ChatGPT 5.0 to its list of supported LLMs.

![ChatGPT 5.0 on a list of supported LLMs](502_ai_connector_gpt_50.png "ChatGPT 5.0 on a list of supported LLMs")

[[= release_note_entry_end() =]]

[[= release_note_entry_begin("Discounts " + version, '2025-08-05', ['LTS Update', 'Commerce']) =]]

#### Discount indexing

Discounts now allow scheduling a mass re-indexing of discounted product catalog prices at the most convenient time by using the Ibexa Messenger package.
Ibexa Messenger is a customization of the Symfony Messenger package, created to adjust it to [[= product_name =]]'s needs.
Once properly configured, it monitors your system resources to trigger price re-indexing without causing performance-based disruptions.

[[= release_note_entry_end() =]]

[[= release_note_entry_begin("Ibexa DXP " + version, '2025-09-09', ['Headless', 'Experience', 'Commerce', 'New feature']) =]]

> #### Collaboration

Reviewer 2: Collaboration appears two times.

With real-time collaboration, your teams can work on the same project at the same time without overwriting each other’s work, and focus mode hides all the distractions.  

#### Improvements to notifications

An improved notifications system is now more intuitive.
Developers can now create and configure their own notification types, while users can now browse through a list of notifications, where they can either act on them or dismiss them.

![A searchable notifications list](502_notifications_screen.png "A searchable notifications list")

#### Developer experience

##### New packages

> The following packages have been introduced in Ibexa DXP v4.6.24:

Reviewer 2: Is it complete? If just one, doesn't need to be in the list, just in one sentence.

- ibexa/messenger

##### PHP API

> The PHP API has been enhanced with the following new classes:

Reviewer 2: Is it complete?

Reviewer: Please use the list from https://github.com/ibexa/documentation-developer/pull/2896#issuecomment-3271125909 , but extract the discounts list to the discount entry

Reviewer 3: The list is available on #2896

##### New version of PHP Storm Plugin

To further improve your experience with Ibexa DXP, a 1.14.0 version of PHP Storm Plugin has been released, which brings the following changes:

>     - Added support for Ibexa DXP v5.0

Reviewer: Please adjust the formatting, it shouldn't be a code block

<img width="953" height="257" alt="Screenshot 2025-09-12 at 11 44 43" src="https://github.com/user-attachments/assets/a251c692-fcec-465f-b9a6-c355f1e9dac0" />

The following additions were made to the Discounts PHP API:

??? note "Events"

>     - [`Ibexa\Contracts\AdminUi\Event\ResolveVersionPreviewUrlEvent`](/api/php_api/php_api_reference/classes/Ibexa-Contracts-AdminUi-Event-ResolveVersionPreviewUrlEvent.html)

Reviewer: This is for AdminUI, please move it to the main product section
Discounts is everything with Ibexa\Contracts\Discounts and Ibexa\Contracts\DiscountsCodes
